### PR TITLE
Dg diagrams gabriel

### LINKED ISSUE #161: Update DG for Edit Customer

### LINKED ISSUE #164: Update DG for Add Delivery

### PR DESCRIPTION
**Pull request by Gabriel4357**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -321,6 +324,103 @@ The following sequence diagram shows how the `login` command works:
 
 <puml src="diagrams/UserLogoutSequenceDiagram.puml" alt="UserLogoutSequenceDiagram" />
 
+//Can state somewhere that all commands need to be logged in before can use
```
Comment (zhonghan721) [inline docs/DeveloperGuide.md]: Currently appears in the markdown file. Perhaps can include the check of login status within the activity/sequence diagrams and description?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -321,6 +324,103 @@ The following sequence diagram shows how the `login` command works:
...
+The `customer edit` command is used to edit an existing Customer with at least one of the information fields
+specified by the user, namely the customer's `Name`, `Phone`, `Email` or/and `Address`.
+
+The format for the `customer edit` command can be found [here](UserGuide.md#login).
+
+**Feature details:**
+
+1. The user specifies the customer id of the `Customer` to be edited, followed by at least one of the information
+fields to be edited,`Name`, `Phone`, `Email` or/and `Address`.
+e.g.(`customer edit 1 --name John --phone 92149601)
```
Comment (zhonghan721) [inline docs/DeveloperGuide.md]: There seems to be a missing ` here and below, for delivery add as well.

**Review by zhonghan721 — COMMENTED** [review]
Some general comments

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Gabriel4357) [reply docs/DeveloperGuide.md]: ok lets discuss with the rest and kiv

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (Gabriel4357) [reply docs/DeveloperGuide.md]: ooo good catch niceee

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -603,6 +606,101 @@ The following sequence diagram shows how the `login` command works:
...
 <puml src="diagrams/CustomerAddSequenceDiagram.puml" alt="CustomerAddSequenceDiagram" />
 
+### Customer Edit Command
+
+**Overview:**
+
+The `customer edit` command is used to edit an existing Customer with at least one of the information fields
+specified by the user, namely the customer's `Name`, `Phone`, `Email` or/and `Address`.
+
+The format for the `customer edit` command can be found [here](UserGuide.md#login).
```
Comment (jianyangg) [inline docs/DeveloperGuide.md]: Should the link be linked to the customer edit feature in the UserGuide instead i.e., `UserGuide.md#edit`? Not sure about the exact reference.

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -603,6 +606,101 @@ The following sequence diagram shows how the `login` command works:
...
+
+### Delivery Add Command
+
+**Overview:**
+
+The `delivery add` command is used to add a new Delivery with all the given information fields
+specified by the user, namely the delivery's `DeliveryName`, customer id of a `Customer` and `DeliveryDate`. All 
+fields are compulsory.
+
+The format for the `delivery add` command can be found [here](UserGuide.md#login).
```
Comment (jianyangg) [inline docs/DeveloperGuide.md]: Same as above, need to update the UserGuide link.

**Comment by Gabriel4357** [comment]
CustomerEditActivityDiagram

CustomerEditDiagram

DeliveryAddActivityDiagram

DeliveryAddDiagram

**Review by jianyangg — APPROVED** [review]
Few nits to fix, otherwise LGTM.

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (Gabriel4357) [reply docs/DeveloperGuide.md]: Done

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (Gabriel4357) [reply docs/DeveloperGuide.md]: yep ok thanks

**Pull request merged**